rai/client_test.go

```diff
@@ -326,10 +326,10 @@ func TestExecuteAsync(t *testing.T) {
 	assert.Nil(t, err)
 
 	expectedResults := []ArrowRelation{
-		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", []interface{}{1., 2., 3., 4., 5.}},
-		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", []interface{}{1., 4., 9., 16., 25.}},
-		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", []interface{}{1., 8., 27., 64., 125.}},
-		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", []interface{}{1., 16., 81., 256., 625.}},
+		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", [][]interface{}{{1., 2., 3., 4., 5.}}},
+		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", [][]interface{}{{1., 4., 9., 16., 25.}}},
+		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", [][]interface{}{{1., 8., 27., 64., 125.}}},
+		ArrowRelation{"/:output/Int64/Int64/Int64/Int64", [][]interface{}{{1., 16., 81., 256., 625.}}},
```

Comment:
I don't understand the change in this commit. There's always going to be only one table per relationId, why does this need to be an array of arrays now?

Reply:
not when multiple results have the same relation id, for example `(1,200); (2,300)` would output
```
[
    {
        "RelationID": "/:output/Int64/Int64",
        "Table": [
            [
                1,
                2
            ],
            [
                200,
                300
            ]
        ]
    }
]
```
the old behavior could output multiple Arrow relations with similar relation id :thinking:

Reply:
oh sorry that's actually wrong I need to update the test :sweat_smile:

Reply:
ohhhh wait i see, is it because it's an array of _columns_?